client/js/components/map/admin/LayerSettings.vue

```diff
@@ -86,7 +86,30 @@
       @input="filterMatrixSetByLayers"
       @selectAll="selectAllLayers"
       @deselectAll="deselectAllLayers"
-    />
+    >
+
+      <template #tag="{ props }">
+        <span class="multiselect__tag">
+          {{ props.option.label }}
+          <dp-contextual-help
```

Comment:
I'm not sure: can this be accessed via keyboard?

Reply:
Which warnings?
I think we should always aim to make things accessible, also in the admin area. I didn't test it, so I wasn't sure if the help text could also be toggled via keyboard (I think it should be possible, but I don't consider this a blocker).